### DISCUSSION TITLE
Publish in Huawei App Gallery

### DISCUSSION THREAD

**Discussion by n3h3m** [iSoron/uhabits#583] [Feature Requests]
I'm using my Huawei P40 pro. This app works flawlessly in P40. It's remarkable since few apps will fail and won't open at all even though the app wouldn't require with Google Play Services. 

Since the app is already working fine in Huawei devices, take a moment to publish in App Gallery since it will reduce the hesitation for other people to install as `.apk`

**Comment by hiqua** [comment]
Is installing FDroid an option?

**Comment by iSoron** [comment]
I will leave this issue open as a discussion. If there is enough demand for it, I might do it. Otherwise, I would recommend installing it from F-Droid instead of downloading the APK directly, as @hiqua suggested.

**Comment by cwerner1** [comment]
+1

The advantage of publishing the app in the App Gallery is that it can generate a large number of users, because not all users have F-Droid installed.

I installed it via F-Droid myself.
